tests/unit/CommonMarkConverterTest.php

```diff
@@ -37,7 +37,7 @@ public function testConfigOnlyConstructor()
     public function testEnvironmentAndConfigConstructor()
     {
         $config = ['foo' => 'bar'];
-        $mockEnvironment = $this->getMock('League\CommonMark\Environment');
+        $mockEnvironment = $this->createMock('League\CommonMark\Environment');
```

Comment:
As we need to keep compatibility with `PHPUnit 4`, `createMock` shouldn't be used. Use `getMockBuilder($class)->getMock()` instead.